src/adapters/notifications/NotificationCenterAdapter.ts

```diff
@@ -292,6 +310,59 @@ export class NotificationCenterAdapter implements NotificationAdapterInterface {
     });
   }
 
+  async projectBoosted(params: {
+    project: Project;
+    userId: number;
+  }): Promise<void> {
+    const { project, userId } = params;
+    const projectOwner = project.adminUser as User;
+
+    return sendProjectRelatedNotification({
+      project,
+
+      // With adding trackId to notification, notification-center would not create new notification
+      // If there is already a notification with this trackId in DB
+      trackId: generateTrackId({
+        userId,
+        projectId: project.id,
+        action: 'boostProject',
+      }),
+      eventName: NOTIFICATIONS_EVENT_NAMES.PROJECT_BOOSTED,
+      sendEmail: true,
+      segment: {
+        analyticsUserId: projectOwner.segmentUserId(),
+        anonymousId: projectOwner.segmentUserId(),
+        payload: getSegmentProjectAttributes({
+          project,
+        }),
+      },
+    });
+  }
+
+  async projectBoostedBatch(params: {
+    projectIds: number[];
+    userId: number;
+  }): Promise<void> {
+    const { userId, projectIds } = params;
+    const projects = await Promise.all(
+      projectIds.map(projectId => findProjectById(projectId)),
+    );
+    projects.forEach(project => {
+      sendProjectRelatedNotificationsQueue.add({
+        project: project as Project,
+        eventName: NOTIFICATIONS_EVENT_NAMES.PROJECT_BOOSTED,
+
+        // With adding trackId to notification, notification-center would not create new notification
+        // If there is already a notification with this trackId in DB
+        trackId: generateTrackId({
+          userId,
+          projectId: project?.id as number,
+          action: 'boostProject',
+        }),
+      });
+    });
```

Comment:
let's instead of duplicating code here, reuse `projectBoosted` method with changes I mentioned above (passing projectId instead of project)

Reply:
Yeah  thanks for your suggestion it's cleaner, Done.